.github/workflows/reusable-python-publish_to_artifactory.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish condan package to Artifactory 
+
+on:
+  workflow_call:
+    inputs:
+        package_name:
+            description: 'Name of the package to build'
+            required: true
+            type: string
+        python_ver:
+            description: 'Python version to use (eg: 3.10)'
+            required: true
+            type: string
+        version_tag:
+            description: 'Version tag of the package to build'
+            required: true
+            type: string
+    secrets:
+        EXTRA_REPO_TOKEN:
+            description: 'Bearer token for the extra repository on JFrog Artifactory'
+            required: true
+
+defaults:
+    run:
+        shell: 'bash -l {0}'
+
+jobs:
+    release_on_artifactory:
+        name: Release package on Artifactory
+        runs-on: ubuntu-latest
+        env:
+            PYTHONUTF8: 1
+            CONDA_CHANNEL_PRIORITY: strict
+            PIP_NO_DEPS: 1 # all dependencies are installed from conda
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+            -   name: Setup CONDA_LOCK_ENV_FILE
+                run: |
+                    os_conda=$( [[ $RUNNER_OS == "Linux" ]] && echo "linux" || ([[ $RUNNER_OS == "Windows" ]] && echo "win" || echo "osx"))
+                    
+                    echo "CONDA_LOCK_ENV_FILE=environments/py-${{ inputs.python_ver }}-${os_conda}-64-dev.conda.lock.yml" >> $GITHUB_ENV
+            -   uses: MiraGeoscience/CI-tools/.github/actions/reusable-python-setup_conda@main
+                name: Setup conda env
+                with:
+                  python_ver: ${{ inputs.python_ver }}
+            -   name: Install dependencies
+                run: |
+                    conda install -y grayskull
+                    conda install -y conda-build
+                    conda install -y conda-verify
+            -   name: Generate conda recipe
+                run: |
+                    tar czf ../${{inputs.package_name}}.tar.gz .
+                    grayskull pypi ../${{ inputs.package_name }}.tar.gz
+            -   name: Build package
+                id: build_package
+                run: |
+                    conda build ${{ inputs.package_name }}
```

Comment:
**question:** does this look for the recipe generated by greyskull?

Reply:
Yes indeed. 
Grayskull generate a ``${{ inputs.package_name}}/meta.yml`, and `conda build` gonna check in the directory name `${{ inputs.package_name}}` for a `meta.yml` file. If it found one, it gonna try to build the conda recipe.